pyproject.toml

```diff
@@ -47,25 +46,30 @@ dependencies = [
   "aacgmv2",
   "numpy",
   "pandas",
+  "scipy",
   "xarray"]
 
 [project.optional-dependencies]
 apex = [ "apexpy" ]
 test = [
-  "coveralls < 3.3",
+  "coverage[toml]",
   "flake8",
   "flake8-docstrings"
 ]
 doc = [
+  "myst_parser",
+  "numpydoc",
+  "pyproject-parser",
   "sphinx",
-  "sphinx_rtd_theme >= 1.2.2,<2.0.0"
+  "sphinx-autoapi",
+  "sphinx_rtd_theme"
 ]
 
 [project.urls]
-# Documentation = "https://pyintensityfeatures.readthedocs.io/en/latest/"
-# Source = "https://github.com/aburrell/pyIntensityFeatures"
-# Tracker = "https://github.com/aburrell/pyIntensityFeatures/issues"
-# Download = "https://github.com/aburrell/pyIntensityFeatures/releases"
+Documentation = "https://pyintensityfeatures.readthedocs.io/en/latest/"
```

Comment:
Go create a RTD page and check that this is the correct link.  Also integrate RTD unit test.